src/CMakeLists.txt

```diff
@@ -161,6 +152,7 @@ set(SM_NAME_RELWITHDEBINFO "StepMania-release-symbols")
 # Configure generated files here.
 configure_file("${SM_XCODE_DIR}/Info.plist.in.xml" "${SM_XCODE_DIR}/Info.StepMania.plist")
 configure_file("${SM_XCODE_DIR}/plistHelper.in.hpp" "${SM_XCODE_DIR}/plistHelper.hpp")
+configure_file("${SM_SRC_DIR}/verstub.in.cpp" "${SM_SRC_DIR}/verstub.cpp")
```

Comment:
Silly question: how do you invoke this at each build? configure_file only runs on Makefile generation, hence my ungodly inception of calling cmake from cmake.